internal/pkg/cli/env_deploy.go

```diff
@@ -42,6 +50,50 @@ type deployEnvOpts struct {
 	unmarshalManifest func(in []byte) (*manifest.Environment, error)
 }
 
+func newEnvDeployOpts(vars deployEnvVars) (*deployEnvOpts, error) {
+	sessProvider := sessions.ImmutableProvider(sessions.UserAgentExtras("env deploy"))
+	defaultSess, err := sessProvider.Default()
+	if err != nil {
+		return nil, err
+	}
+	store := config.NewSSMStore(identity.New(defaultSess), ssm.New(defaultSess), aws.StringValue(defaultSess.Config.Region))
+	ws, err := workspace.New()
+	if err != nil {
+		return nil, fmt.Errorf("new workspace: %w", err)
+	}
+	if err != nil {
+		return nil, fmt.Errorf("initiate a new environment deployer: %w", err)
+	}
```

Comment:
How does second if will execute if there is an error while creating a new workspace?
And I think second if looks incorrect as it says _initiate a new environment deployer_ which doesn't seem like happening around this code and looks like happening down on line 79

Reply:
I forgot to remove the second error handling 😄

Reply:
removed and addressed!